Documentation for Snapshot 2.0 (#518)
IBM/ibm-block-csi-driver: docs/content/release_notes/limitations.md

## I/O group limitations

**Note:** I/O group configuration is only supported for use with IBM Spectrum Virtualize family storage systems.

## Snapshot function limitations

**Important:** Snapshot function support is currently only Alpha support.

- Snapshot function is only supported for use with IBM Spectrum Virtualize family storage system versions 8.5.1 or higher. For more information, see **Product overview** > **Technical overview** > **Volume groups** > **Snapshot function** within your Spectrum Virtualize product documentation on [IBM Documentation](https://www.ibm.com/docs).

> - Both source and target PVCs (in a source PVC to snapshot to target PVC scenario) but have the same space efficiency set within their storage classes. If the space efficiency is set differently, the target PVC creation fails.

Reviewer: I dont understand the `but`

Author: ** typo - must.
 fixed.

> - A snapshot using the Snapshot function cannot be created with space efficiency set. If the VolumeSnapshotClass has the SpaceEfficiency parameter set along with the snapshot flag enabled, the snapshot creation fails.

Reviewer 2: ```suggestion
- A snapshot using the Snapshot function cannot be created with space efficiency set. If the `VolumeSnapshotClass` has the `SpaceEfficiency` parameter set along with the `virt_snap_func` flag enabled, the snapshot creation fails.
```

Author: updated

- In very rare cases, there may be leftover or undeleted volumes.

> - A snapshot using the Snapshot function must be created within the same PVC or child pool as the original PVC.

Reviewer 2: ```suggestion
- A snapshot using the Snapshot function must be created within the same pool or child pool as the original PVC.
```

Author: @ArbelNathan  please confirm. I already changed per your last comment
![image](https://user-images.githubusercontent.com/54570654/175254078-3d5edadc-9e74-422d-a268-1a3b10540e06.png)

Reviewer: I approve Ori's change

Author: thanks - done

> - A snapshot using the Snapshot function must be created within the same PVC volume or child pool as the original PVC.

Reviewer: ```suggestion
- A snapshot using the Snapshot function must be created within the same PVC pool or child pool as the original PVC.
```?

Author: fixed

> - Snapshots do not have unique IDs (UUIDs). As a result, if a snapshot with Snapshot function is deleted another snapshot can get the same ID number.

Author: thanks - I checked within SV documentation - unique identifier = UID. i'll update accordingly

> - Snapshots do not have unique IDs (UUIDs). As a result, if a snapshot with Snapshot function is deleted another snapshot can get the same ID number.

Author: done

> - Any object that is linked in any way (for example, a clone or a snapshot) must have the same definition of snapshot support. For example, a clone cannot be created with FlashCopy mapping enabled from a PVC with an existing Snapshot function connection.

Reviewer 2: ```suggestion
- Any object that is linked in any way (for example, a clone or a snapshot) must have the same definition of snapshot support. For example, a clone cannot be created with `virt_snap_func` disabled, from a PVC with an existing Snapshot function connection.
```

>     **Note:** FlashCopy mapping (`fcmap`) and Snapshot function cannot be used together on the same volume however they can be used on different volumes within the same storage system.

Reviewer 2: ```suggestion
    **Note:** FlashCopy mapping (`fcmap`) and Snapshot function cannot be used together on the same volume. however, they can be used on different volumes within the same storage system.
```
